src/domain/event/Event.tsx

```diff
@@ -56,6 +87,51 @@ const Event: FunctionComponent = () => {
 
   if (!data?.event) return <div>No event</div>;
 
+  const optionsDates = data.event.occurrences.edges
+    .map(occurrence => {
+      return occurrence?.node?.id && occurrence.node.time
+        ? {
+            value: formatTime(newMoment(occurrence.node.time), 'YYYY-MM-DD'),
+            label: formatTime(
+              newMoment(occurrence.node.time),
+              DEFAULT_DATE_FORMAT
+            ),
+            key: occurrence.node.id,
+          }
+        : { key: '', label: '', value: '' };
+    })
+    .filter((v, i, a) => a.findIndex(t => t.value === v.value) === i);
+
+  const optionsTimes = data.event.occurrences.edges
+    .map(occurrence => {
+      return occurrence?.node?.id && occurrence.node.time
+        ? {
+            value: formatTime(newMoment(occurrence.node.time), 'HH:mm'),
+            label: formatOccurrenceTime(
+              occurrence.node.time,
+              data.event?.duration || null
+            ),
+            key: occurrence.node.id,
+          }
+        : { key: '', label: '', value: '' };
+    })
+    .filter((v, i, a) => a.findIndex(t => t.value === v.value) === i)
+    .sort((a, b) => {
+      return a.label && b.label
```

Comment:
Do we need this first check? Also this nested conditional operand is bit hard to read.